### DISCUSSION TITLE
I am having problems with EotE DLC

### DISCUSSION THREAD

**Discussion by NinG4** [Raicuparta/nomai-vr#496] [Q&A] [answered]
Hello ! I don't know if the project is still ongoing or not, I just had my headset recently and found the mod.
I have OW and its DLC installed on Epic Games Store but it disappears without reason when I switch to NomaiVR, and I can't play it (the museum is still in "Coming soon" and nothing is triggered on this side.
But when I switch back to the normal version, I have the DLC. I tried reinstalling everything 2 or 3 times and it doesn't change a thing (even with anterior version).
I really don't know what to do to fix that, can you help me please ?

**Comment by Raicuparta** [comment, marked as answer]
Try these:

Try opening the epic games store in the background, and then start the game via the mod manager.

Try opening the game via the epic games store, after having opened the game in VR at least once.

Confirm that this switch is **off**
![imagen](https://user-images.githubusercontent.com/3955124/147685617-e0a72f24-839c-48ee-a711-f0109a72ab74.png)

**Reply by NinG4** [reply]
It worked, I never realized OWMM replaced my game in the Epic Games Store, and I always started it through OWMM, I feel stupid but happy :)
Thanks for your help and speed !
Have a great day/night

**Reply by Raicuparta** [reply]
So you started it via the epic games store, yes? Or did you start via the mod manager with the epic store running in the background? The mod manager is actually supposed to work in this scenario, so I'm trying to understand why it didn't work for you.

**Reply by NinG4** [reply]
Yes I started it with the Epic store. I was always starting it from OWMM before so it may not have detected the license for the DLC because of that.
